lyndapham/step/portfolio/src/main/webapp/essays.html

```diff
@@ -4,28 +4,33 @@
     <meta charset="UTF-8">
     <title>My Essays</title>
     <link rel="stylesheet" href="style.css">
-    <!-- <script src="script.js"></script> -->
+    <script src="script.js"></script>
   </head>
   <body>
-        <main>
-            <h1>Lynda's Thoughts</h1>
-            <div class="content">
-                <div class="pdf">
-                    <h3>Driving</h3>
-                    <object width="360" height="270" data="pdfs/Driving.pdf"></object>
-                </div>
+      <div id="nav-bar">
```

Comment:
The indentation is still inconsistent. Notice how above the indentation is 2 spaces but here is 4 spaces

Reply:
Fixed! Sorry about that.